pkg/tfgen/generate_python.go

```diff
@@ -457,6 +457,17 @@ func (g *pythonGenerator) emitResourceType(mod *module, res *resourceType) (stri
 	if res.IsProvider() {
 		baseType = "pulumi.ProviderResource"
 	}
+
+	if !res.IsProvider(){
+		deprecationMessage := g.getDeprecationDetails(res.info)
+		if deprecationMessage != "" {
+			w.Writefmtln("warnings.warn(")
+			w.Writefmtln("    %s,", deprecationMessage)
+			w.Writefmtln("    DeprecationWarning")
+			w.Writefmtln(")")
+		}
+	}
+
 	// Produce a class definition with optional """ comment.
```

Comment:
avoid comment if the type is deprecated.  or, the comment should just have the deprecation message in it.